driver-scala/src/it/scala/org/mongodb/scala/RequiresMongoDBISpec.scala

```diff
@@ -28,6 +28,25 @@ import scala.util.{ Properties, Try }
 
 trait RequiresMongoDBISpec extends BaseSpec with BeforeAndAfterAll {
 
+  private object GlobalState {
```

Comment:
This may be the completely wrong Scala idiom for what I'm trying to accomplish, which is to make all these values singletons.  It does seem to work, but happy for any suggestions to make it more idiomatic.